examples/.examples-ci.yml

```diff
@@ -0,0 +1,77 @@
+examples:
+  stage: examples
+  needs: []
+  parallel:
+    matrix:
+      - DOTNET_VERSION: ['6.0']
```

Comment:
do you want 7 and 8 here too? or just 8 here?

Reply:
Tried, but the examples actually fail to run on anything but 6.0.

Reply:
Failed to run or to build? Have you filed a ticket for it?

Reply:
Okay I should've said failed to build, because they all target .NET 6 so that's the version that must be installed in order to build. If we multi-targeted them then we would need multiple SDK versions installed at once.